pipeline/runtime/step_executor.go

```diff
@@ -38,6 +39,7 @@ type StepExecutor struct {
 	engine     *engine.Engine
 	mu         sync.Mutex
 	stepStatus map[string]StepStatus
+	stepOutput map[string]*StepOutput
```

Comment:
Can we use stepLogOutput instead of stepOutput? step output in CIE refers to output variables.

Reply:
Well, we are capturing what a step writes to standard output. So, technically, it should be called 'output'. Output variables should be called outputVars or something like that.

Reply:
Logs are captured on standard error as well. So, by that reasoning it should be stepOutput+Error.

Step output can refer to couple of things e.g. output variables, logs etc. Renaming it to avoid confusion.